src/Templates/NServiceBusWindowsService/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using System.ServiceProcess;
+using System.Threading.Tasks;
+
+namespace NServiceBusWindowsService
+{
+    internal static class Program
```

Comment:
I prefer to always be explicit. I don't like reading code and having to mentally map back to the specific defaults for each type of member. E.g. for classes it's `internal`, for fields and method it's `private`, etc.

Incidentally, this is also the [preference of the dotnet org:](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

> 5. We always specify the visibility, even if it's the default (i.e.
   `private string _foo` not `string _foo`). Visibility should be the first modifier (i.e. 
   `public abstract` not `abstract public`).

However, if @Particular/host-maintainers prefer not to restate the defaults then I'll remove them.

Reply:
My vote is to remove them, primarily because none of the default Microsoft templates include them, and I'd like to see our stuff match the default templates as much as possible.

Reply:
OK, that's a reasonable argument. I'll remove them.

Reply:
Can we come up with a company wide approach to this? I'm too old to remember settings like this on a repo basis. And no I won't install R# 😃

Reply:
I'm generally 👎 to company wide standards, but I do think the code should tell us what the standard for a repo is, without imposing a local tooling choice (e.g. R#). It could be done via EditorConfig (now built into VS), Roslyn analyzers, etc.